README.md

```diff
@@ -185,7 +185,8 @@ Possible options are below. If left blank, they will be ignored and defaults use
 
 Argument | Description
 -------- | -----------
-cluster_name | The name of the cluster (not the ARN). eg 'my-super-cluster'. Required
+cluster_name | The name of the cluster (not the ARN). eg 'my-super-cluster'. Required (unless using spot_request_id)
+spot_request_id | The spot request ID you are connecting to. eg 'sfr-abcdef'. Required (unless using cluster_name)
```

Comment:
I wonder if specifying an ASG would be more flexible and allow for use for spot and non-spot?

Reply:
While spot fleets have auto-scaling on them, I don't believe there is an actual ASG resource associated with them. We could just accept either an ASG name or spot fleet ID and check if it starts with `sfr` but it feels a touch icky and then it might hurt if AWS ever changes what their spot fleet ID's look like (unlikely but hey)

Reply:
Yeah agreed! Sorry I didn't realise that there was no ASG resource, carry on :+1: